### ISSUE TITLE
ModuleNotFoundError: No module named 'rdflib.plugins.parsers.pyRdfa' for RDFLib v5.0.0

### ISSUE THREAD

**Issue by tonal** [scrapinghub/extruct#131]
```
>>> import extruct
RDFLib Version: 5.0.0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/.pyenv/versions/3.6.10/envs/project/lib/python3.6/site-packages/extruct/__init__.py", line 1, in <module>
    from ._extruct import SYNTAXES, extract
  File "/home/user/.pyenv/versions/3.6.10/envs/project/lib/python3.6/site-packages/extruct/_extruct.py", line 5, in <module>
    from extruct.rdfa import RDFaExtractor
  File "/home/user/.pyenv/versions/3.6.10/envs/project/lib/python3.6/site-packages/extruct/rdfa.py", line 14, in <module>
    from rdflib.plugins.parsers.pyRdfa import pyRdfa as PyRdfa, Options, logger as pyrdfa_logger
ModuleNotFoundError: No module named 'rdflib.plugins.parsers.pyRdfa'
```

**Comment by dnnsthnnr** [comment]
@tonal #132  I had the same issue. Just install the rdflib==4.2.2

**Comment by lopuhin** [comment]
Thanks for report @dnnsthnnr and @tonal . For now pinning rdflib is indeed the best solution, but we'll check if we can update extruct to work with a newer version as well.

**Comment by lopuhin** [comment]
from https://rdflib.readthedocs.io/en/stable/upgrade4to5.html

> The RDFa and Microdata format RDF parsers were removed from rdflib. There are still other python libraries available to implement these parsers.

And indeed the parsers we use were removed in https://github.com/RDFLib/rdflib/pull/828/

So it looks like we don't be able to use a newer rdflib version, short-term solution would be to pin it, and long-term to check these other libraries.

**Comment by lopuhin** [comment]
This is released in 0.9.0
